abb_resources/urdf/common_colours.xacro

```diff
@@ -38,6 +38,9 @@
   <!-- RAL 6002 leaf green -->
   <xacro:property name="colour_abb_leaf_green" value="0.1764706 0.3529412 0.1529412 1" />
 
+  <!-- RAL 6003 olive green -->
+  <xacro:property name="colour_abb_olive_green" value="0.3137254 0.3254902 0.2352941 1" />
```

Comment:
Could you please change this to the following style (ignore the actual values, it's a copy-paste):

```xml
value="${ 61/255} ${ 61/255} ${ 61/255} 1.0"
```

Using `xacro`'s built-in math support makes sure we use the integer values from the RGB triplet while also using maximum precision available in the conversion.

I know the other `property`s don't do this, but they should be converted.

Reply:
Updated!